src/Mono.Android/Mono.Android.targets

```diff
@@ -2,7 +2,7 @@
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <UsingTask AssemblyFile="..\..\bin\Build$(Configuration)\xa-prep-tasks.dll"   TaskName="Xamarin.Android.BuildTools.PrepTasks.ReplaceFileContents" />
   <Import Project="..\..\build-tools\scripts\XAVersionInfo.targets" />
-  <Import Project="Mono.Android.projitems" />
```

Comment:
Shouldn't this PR also *delete* `src/Mono.Android/Mono.Android.projitems`?

Reply:
Please, do not review these changes yet. :)